pyphare/pyphare/pharesee/hierarchy/hierarchy_compute.py

```
@@ -0,0 +1,94 @@
+
+def rename(hierarchy, names):
+    from .hierarchy_utils import compute_hier_from
+
+    return compute_hier_from(compute_rename, hierarchy, new_names=names)
+
+
+def compute_rename(patch, **kwargs):
+    new_names = kwargs["new_names"]
+    pd_attrs = []
+
+    for new_name, pd_name in zip(new_names, patch.patch_datas):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)